simplq/src/styles/joinPage.module.scss

```diff
@@ -13,3 +17,39 @@
     margin: 1rem;
   }
 }
+.header-title {
+  margin: auto;
+  .sub-header {
+    display: flex;
+    font-family: Pacifico;
+    font-size: 0.8rem;
+    font-weight: 300;
+    color: #3a3768;
+    align-items: center;
+  }
+}
+
+.header-bar {
+  @include background-color();
+  padding: 0.5rem;
+  // padding-top: 4rem;
+  
+  display: flex;
+  flex-wrap: wrap;
+  .simplq-logo{
+    margin-top: 35px;
+  }
+  .header {
+    @include standard-header();
+    font-size: 2.7rem;
+    margin-bottom: 0;
+  }
+}
+
+@media screen and (max-width: 1000px) {
+  .header-bar {
```

Comment:
rem here too please.

Reply:
Okkk I'll change it soon , I'll keep that in mind to use rem. 
And that name and description is looks perfect (centered) in my machine what could be the issue. The screenshot that you have posted has very different than mine .

I'll try to figure it out

Reply:
I think px would be fine for media queries , for general rem would be fine

Reply:
Yeah I suppose that's fine.